tad/api/routes/root.py

```diff
@@ -1,11 +1,17 @@
 from fastapi import APIRouter, Request
-from fastapi.responses import HTMLResponse
+from fastapi.responses import FileResponse, HTMLResponse
 
-from tad.api.deps import templates
+from tad.core.config import settings
+from tad.repositories.deps import templates
 
 router = APIRouter()
 
 
 @router.get("/")
 async def base(request: Request) -> HTMLResponse:
     return templates.TemplateResponse(request=request, name="root/index.html")
+
+
+@router.get("/favicon.ico", include_in_schema=False)
```

Comment:
It would be better if we redirect the favicon loading to the static url with an html header:
`
 <link rel="icon" type="image/x-icon" href="/images/favicon.ico">
`

Reply:
I'm ok with doing that later. I did consider it, and then figured I was happy having a favicon always work from an expected default location.